lib/bot.js

```diff
@@ -335,6 +347,44 @@ class Bot {
     return null;
   }
 
+  findWebhook(ircChannel) {
+    const discordChannelName = this.invertedMapping[ircChannel.toLowerCase()];
+    if (discordChannelName) {
+      const webhook = this.webhooks[discordChannelName];
+      if (webhook) {
+        return webhook;
+      }
+    }
+    return null;
+  }
+
+  getDiscordAvatar(nick, channel) {
+    const guildMembers = this.findDiscordChannel(channel).guild.members;
+    const findByNicknameOrUsername = caseSensitive =>
+      (member) => {
+        if (caseSensitive) {
```

Comment:
(Yes but I suggested [here](https://github.com/reactiflux/discord-irc/pull/230#discussion_r125168767) the logic for how to implement it. :P)